components/VM_Arm/src/main.c

```diff
@@ -927,15 +945,16 @@ static int load_linux(vm_t *vm, const char *kernel_name, const char *dtb_name, c
     return 0;
 }
 
-void parse_camkes_linux_attributes(void)
+void parse_camkes_vm_attributes(void)
 {
-    linux_ram_base = strtoul(linux_address_config.linux_ram_base, NULL, 0);
-    linux_ram_paddr_base = strtoul(linux_address_config.linux_ram_paddr_base, NULL, 0);
-    linux_ram_size = strtoul(linux_address_config.linux_ram_size, NULL, 0);
-    linux_ram_offset = strtoul(linux_address_config.linux_ram_offset, NULL, 0);
-    dtb_addr = strtoul(linux_address_config.dtb_addr, NULL, 0);
-    initrd_max_size = strtoul(linux_address_config.initrd_max_size, NULL, 0);
-    initrd_addr = strtoul(linux_address_config.initrd_addr, NULL, 0);
+    ram_base = strtoul(vm_address_config.ram_base, NULL, 0);
+    ram_paddr_base = strtoul(vm_address_config.ram_paddr_base, NULL, 0);
+    ram_size = strtoul(vm_address_config.ram_size, NULL, 0);
+    ram_offset = strtoul(vm_address_config.ram_offset, NULL, 0);
+    dtb_addr = strtoul(vm_address_config.dtb_addr, NULL, 0);
+    initrd_max_size = strtoul(vm_address_config.initrd_max_size, NULL, 0);
+    initrd_addr = strtoul(vm_address_config.initrd_addr, NULL, 0);
+    entry_addr = strtoul(vm_address_config.kernel_entry_addr, NULL, 0);
```

Comment:
I wonder if it's better to put the platform default magic here, because we can differentiate better
- parameter not set -> do platform default magic, but print a warning that this is deprecated and the entry address must be given explicitly
- parameter is not a string that contains an integer -> error
- parameter is 0 -> use this value
- parameter is an address -> use this value

Reply:
I added a function that parses the entry_addr parameter. Default is set to `-1`, which won't trigger an error on build, but will print a warning that it was calculated.

Reply:
Thanks, that looks really nice now.